apps/core/src/utils/transaction/isMigrationTransaction.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import type {
+    IotaTransaction,
+    IotaTransactionBlockResponse,
+    MoveCallIotaTransaction,
+} from '@iota/iota-sdk/client';
+import { STARDUST_PACKAGE_ID } from '../../constants';
+
+export function isMigrationTransaction(transaction: IotaTransactionBlockResponse['transaction']) {
+    if (!transaction || transaction.data.transaction.kind !== 'ProgrammableTransaction')
+        return { isMigration: false };
+    const moveCallTxs = transaction.data.transaction.transactions.filter(isMoveCall);
+    const isMigration = moveCallTxs.some(
+        (tx) =>
+            tx.MoveCall.package === STARDUST_PACKAGE_ID &&
+            tx.MoveCall.function === 'extract_assets',
+    );
+
+    return {
+        isMigration,
```

Comment:
why do we return an object with a single boolean instead of just returnining a boolean